File: Lecture004 Solving Pattern Questions (Part-2)/pattern9.py

class solution():
    def pattern(n):
        row=1
        
        while(row<=n):
            col=1
            str=ord('A')+row-1
            while(col<=row):
                print(chr(str),end=" ")
                str+=1
                col+=1
            print()
            row+=1

if __name__ == '__main__':
    ob=solution
    num = int(input("Enter Number = "))
    ob.pattern(num)

File: Lecture004 Solving Pattern Questions (Part-2)/pattern9.java
import java.util.Scanner;

class pattern9 {
    public static void main(String[] args) {
        int n;

        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        n = sc.nextInt();
        sc.close();

        for (int row = 1; row <= n; row++) {

            // print space
            for (int col = n - row; col >= 1; col--) {
                System.out.print(" ");
            }

            // print stars
            for (int col = 1; col <= row; col++) {
                System.out.print("*");
            }

            System.out.println();
        }
    }
}
